addons/captives/stringtable.xml

```diff
@@ -228,22 +235,27 @@
             <German>Spieler können kapitulieren, nachdem sie ihre Waffe geholstert haben</German>
             <Portuguese>Jogadores podem se render depois de guardar sua arma</Portuguese>
             <Hungarian>A játékosok megadhatják magukat a fegyverük elrakása után</Hungarian>
+            <Russian>Игроки могут сдаться в плен после того, как уберут оружие</Russian>
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_name">
             <English>Require surrendering</English>
             <Polish>Wymagaj kapitulacji</Polish>
+            <Russian>Требовать пленения</Russian>
         </Key>
         <Key ID="STR_ACE_Captives_ModuleSettings_requireSurrender_description">
             <English>Require Players to surrender before they can be arrested</English>
             <Polish>Wymagaj od graczy kapitulacji zanim będzie można ich zaaresztować</Polish>
+            <Russian>Требуется, чтобы игрок сдался в плен прежде, чем его можно будет связать</Russian>
         </Key>
         <Key ID="STR_ACE_Captives_SurrenderOnly">
             <English>Surrendering only</English>
             <Polish>Tylko kapitulacja</Polish>
+            <Russian>Только сдавшийся в плен</Russian>
         </Key>
         <Key ID="STR_ACE_Captives_SurrenderOrNoWeapon">
             <English>Surrendering or No weapon</English>
```

Comment:
You are probably right on this one. For some reason I considered "unarmed" to be equally valid in describing a person with holstered or concealed weapon, while in reality of common usage it pretty much equates to "weaponless". Similarly, "armed" has too broad of a sense to effectively convey the opposite meaning of (having a) "weapon drawn".

I guess, we can't get any shorter than the original "no weapon", as English doesn't seem to have any single-word terms applicable specifically to the state in question. Oh, well, whatever, nevermind. :grin: